unlock-app/app/Components/ProtectedContent.tsx

```diff
@@ -0,0 +1,19 @@
+'use client'
+
+import { ReactNode } from 'react'
+import { WalletNotConnected } from '~/components/interface/layouts/index/WalletNotConnected'
+import { useAuth } from '~/contexts/AuthenticationContext'
+
+interface ProtectedContentProps {
+  children: ReactNode
+}
+
+export function ProtectedContent({ children }: ProtectedContentProps) {
```

Comment:
I think we need a more explicit name like `<AuthRequired>`, wdyt?

Reply:
definitely more succint! thank you!